externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/PlanOnlyStatement.scala

```diff
@@ -36,6 +36,11 @@ class PlanOnlyStatement(
   extends SparkOperation(OperationType.EXECUTE_STATEMENT, session) {
 
   private val operationLog: OperationLog = OperationLog.createOperationLog(session, getHandle)
+  private val configExcludes: Option[String] =
+    spark.conf.getOption(OPERATION_PLAN_ONLY_EXCLUDES.key)
+  private val planExcludes: Seq[String] =
+    if (configExcludes.isEmpty) session.sessionManager.getConf.get(OPERATION_PLAN_ONLY_EXCLUDES)
+    else configExcludes.get.split(",").toSeq
```

Comment:
```suggestion
  private val planExcludes: Seq[String] = {
    spark.conf.getOption(OPERATION_PLAN_ONLY_EXCLUDES.key).map( _.split(",").map(_.trim).toSeq)
      .getOrElse(session.sessionManager.getConf.get(OPERATION_PLAN_ONLY_EXCLUDES))
  }
```

Reply:
we need to split the session conf `session.sessionManager.getConf.get(OPERATION_PLAN_ONLY_EXCLUDES)` by comma as well.

Reply:
@cfmcgrady, the return value of the `session.sessionManager.getConf.get(OPERATION_PLAN_ONLY_EXCLUDES)` is string sequence. Thus, the above suggestion has something wrong because you map the `Option`.

Reply:
Yes, you are right, I misread before :)

Reply:
@yaooqinn, does this suggestion is wrong? As below mentioned, the priority of session conf should be higher than spark conf.

Reply:
seesion conf has already been set to spark confs